README.md

```diff
@@ -121,7 +121,7 @@ jobs:
 
 #### Debugging
 
-To debug an issue with the goat, configure the log level to either `ERROR`, `WARN`, `NOTICE`, `VERBOSE`, `DEBUG`, or `TRACE`.
+To debug an issue with the goat, configure the log level to either `ERROR`, `WARN`, `INFO`, or `DEBUG`.
```

Comment:
I cleaned this up to only the log levels we actually use and added debug logging because I needed it to troubleshoot some changes that I was making.